database/migrations/2024_03_18_164714_add_decline_msg_to_checkout_acceptance_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddDeclineMsgToCheckoutAcceptanceTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('checkout_acceptances', function (Blueprint $table) {
+            $table->string('declined_msg')->after('signature_filename')->nullable();
```

Comment:
A smaller comment but any reason we're using `declined_msg` instead of `declined_message`? I'm personally not a big fan of abbreviating variables/columns and `message` isn't that many more keystrokes.

That might be an app-wide question and not restricted to this PR.

Reply:
what about `declined_monosodium_glutamate`? Jk, changes added 😂

Reply:
> `declined_monosodium_glutamate`

Never mind...back to `msg` 😝 

(thanks for the change)